backend/test/workers/jobs.test.js

```diff
@@ -77,3 +79,17 @@ test('sendSampleEmailJob creates job', async t => {
 	const [taskedJob] = await sendSampleEmail.getRepeatableJobs();
 	t.is(taskedJob, undefined);
 });
+
+test('deleteAccountJob creates job', async t => {
+	const job = await deleteAccountJob(user);
+	t.is(job.returnvalue, null);
+	t.is(job.name, '__default__');
+	t.is(job.opts.attempts, QUEUE_ATTEMPTS.DELETE_ACCOUNT);
+	t.deepEqual(job.data, { userID: user.id });
+	t.regex(job.id, /\d+/);
```

Comment:
check eq with `user.id`

Reply:
Must modify all job tests, so this correction will happen at a later stage.